src/Test/StateMachine/Utils.hs

```diff
@@ -0,0 +1,13 @@
+module Test.StateMachine.Utils where
```

Comment:
There's already a module called `Test.StateMachine.Internal.Utils`, perhaps it would make sense to put those things there?

Reply:
Yeah that makes sense. I was trying to make a distinction between these utils for internal use, and the others as a convenience for users. But some of these functions might be useful in both categories, so it may be better instead to put everything in one module used internally and have `Test.StateMachine.Utils` reexport a relevant subset of it.